js/songs/songs.module.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+// As a user, we should be able to see all of our songs in the database.
+// As a user, when we first go to the app, we see a welcome page.
+// Bonus: As a user, when we select a song it starts playing. The world is a better place
+
+(function(){
+  angular
+  .module('songs', ["ngResource"]);
```

Comment:
I would recommend separating `ngResource` onto it's own separate line for readability.